From a4fbfb88f1b7c4f4287d9279767220fae80d26da Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jos=C3=A9=20Bollo?= <jose.bollo@iot.bzh>
Date: Thu, 21 Jan 2016 15:07:29 +0100
Subject: [PATCH] Hack to allow the debugging

This is a temporarily fix to continue debugging
afm-main. This should be removed later.

Change-Id: I2f10f0cb1fce2ee30bd0754ad2e7bc8e2f6513aa
---
 conf/afm-user-daemon.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/user/afm-user-daemon.conf b/conf/user/afm-user-daemon.conf
index 801c7ae..98a3152 100644
--- a/conf/user/afm-user-daemon.conf
+++ b/conf/user/afm-user-daemon.conf
@@ -25,7 +25,7 @@
         </policy>
 
         <policy context="default">
-                <deny own="org.AGL.afm.user"/>
+                <allow own="org.AGL.afm.user"/>
                 <allow send_destination="org.AGL.afm.system"/>
         </policy>
 
-- 
2.1.4

